andela/Titan-Devs
# 163518692 User Can Like Article

#### What does this PR do?
Let authenticated user like an article or dislike an arleady liked article
#### Description of Task to be completed?
- Write tests for article likes
- The article like and dislike
- Fetch article with likes
#### How should this be manually tested?
- Clone the repo and  check out `ft-article-likes-dislike-163518692` branch
- Follow the instructions in `ReadMe`
- Using any `API client tool *POSTMAN*`,
- Create the user and login to get the ```POST /api/v1/users``` and ```POST /api/v1/users/login```
- Create an article using ```POST /api/v1/articles``` and get the ```slug```
- Access `POST /api/v1/articles/:slug/likes` endpoint for liking an article
- Access ```POST /api/v1/articles/:slug/dislikes``` endpoint for disliking the article
- Make sure to pass the token under `request headers`. It should be {`Authorization`: `Bearer <access-token>`} KEY, VALUE PAIR
- Access ```GET /api/v1/articles/:slug/likes``` to fetch article with its likes
#### Any background context you want to provide?
Trying to like article twice, cancel the first like. 
#### What are the relevant pivotal tracker stories?
#163518692
